apps/api/src/bootstrap.ts

```diff
@@ -121,7 +122,8 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   await setupSwagger(app);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Please see [line 93](https://github.com/novuhq/novu/pull/6601/files#diff-11933634457bf1519b686a8747a356d1c21a23aa19c22984b3da6c875919c6c3R93) above, we already have Sentry instrumentation to handle exceptions. We should make sure that we're handling exceptions consistently with a single entry point.

Related: it seems that the latest version of `@sentry/node` removed the `Handlers` export, [see here for more details on migrating](https://github.com/getsentry/sentry-javascript/issues/12007). We should explore this upgrade, it's possible that the newer versions of Sentry have built-in APIs for exception mapping.‏

Reply:
for context: we have synced and tested the change with sentry, the decision is to attempt to match the sentry error id with the uuid if exist (for cloud) besides that this functionality works great and does not damage the sentry functionality